### PR TITLE
Prefix .nav-item styles with .sidebar so not to affect bootstrap tabs elsewhere

### PR DESCRIPTION
**Pull request by adrianwright109**
Addresses #11267

### CODE REVIEW

File: src/Components/Blazor/Templates/src/content/BlazorWasm-CSharp/Client/wwwroot/css/site.css [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
 
 html, body {
     font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
 }
 
+a, .btn-link {
+    color: #0366d6;
+}
+
+.btn-primary {
```
Comment (pranavkm) [inline src/Components/Blazor/Templates/src/content/BlazorWasm-CSharp/Client/wwwroot/css/site.css]: @adrianwright109 could you explain why these were changed? This seems unrelated to the linked issue. We'd prefer an absolute minimal PR that fixes the addressed issue.

**Review by pranavkm — CHANGES_REQUESTED** [review]

File: src/Components/Blazor/Templates/src/content/BlazorWasm-CSharp/Client/wwwroot/css/site.css (hunk 1 above)
Comment (adrianwright109) [reply src/Components/Blazor/Templates/src/content/BlazorWasm-CSharp/Client/wwwroot/css/site.css]: I assumed all 5 `site.css` should be the same so added the missing sections so that they are now all the same.

**Comment by pranavkm** [comment]
@SteveSandersonMS could you have a look?

**Review by SteveSandersonMS — APPROVED** [review]
Looks great. Thanks for sorting out the inconsistencies too!

File: src/Components/Blazor/Templates/src/content/BlazorWasm-CSharp/Client/wwwroot/css/site.css (hunk 1 above)
Comment (SteveSandersonMS) [reply src/Components/Blazor/Templates/src/content/BlazorWasm-CSharp/Client/wwwroot/css/site.css]: @pranavkm Having reviewed this now, I agree it's good to make these changes to bring all the CSS into consistency. If you're OK with that, could you withdraw your "request changes" so I can merge this?
